package.json

```diff
@@ -2,11 +2,15 @@
   "name": "react-responsive-picture",
   "version": "1.0.1",
   "description": "A future-proof responsive image component that supports latest Picture specification",
-  "main": "./dist/index.js",
+  "main": "./lib/index.js",
+  "module": "es/index.js",
+  "jsnext:main": "es/index.js",
   "scripts": {
-    "build": "babel src/ -d dist/ --copy-files",
-    "clean": "rimraf dist/",
-    "dev": "yarn run clean && babel src/ -d dist/ --watch --copy-files",
+    "build": "yarn run build:commonjs && yarn run build:es",
+    "build:es": "babel src -d es",
+    "build:commonjs": "cross-env BABEL_ENV=commonjs babel src -d lib",
+    "clean": "rimraf lib dist es",
+    "dev": "yarn run clean && babel src -d lib --watch",
```

Comment:
This implies that we'll always be building the commonjs version during dev. What's the idea there?